newIDE/app/src/UI/Theme/RosePineTheme/theme.json

```diff
@@ -182,6 +182,18 @@
           "value": "{theme.surface.canvas.background-color.value}"
         }
       }
+    },
+    "home": {
```

Comment:
home changed place only in this theme?

Reply:
Yes, it was ill-configured: the home part was in `events-sheet`